assets/high-contrast-canvas.css

```diff
@@ -8,33 +8,18 @@
   --context-pad-entry-background-color: var(--gray-10);
 }
 
-.dmn-drd-container,
-.bjs-container {
+.djs-container.bio-improved-canvas,
+.dmn-drd-container.bio-improved-canvas {
   background-color: var(--gray-10);
 }
 
-.dmn-drd-container {
+.dmn-drd-container.bio-improved-canvas {
   --dmn-definitions-background-color: white;
   --dmn-definitions-border-color: var(--gray-20);
 }
 
-/* Properties Panel */
-
-.bio-properties-panel-header {
-  background-color: white;
-}
-
-/* Keep forms white */
-.cfp-container .bio-properties-panel-header {
-  background-color: var(--header-background-color);
-}
```

Comment:
Great to get rid of this hack

Reply:
I was surprised to find this. 😅 All web modeler style changes have been moved to web modeler.